requirements.txt

```diff
@@ -16,3 +16,4 @@ repoze.lru
 gevent-websocket==0.9.4
 sphinx
 bumpversion
+cachetools>=2.0.0<3.0.0
```

Comment:
Why the `<3.0.0`? Cachetools [latest version](https://pypi.python.org/pypi/cachetools) seems to be `2.0.0`.
Is it to avoid any unintentional code behaviour changes in the case a new `cachetools` major version is released?